arbor/fvm_layout.cpp

```diff
@@ -29,10 +29,9 @@
 #include "util/unique.hpp"
 #include "util/strprintf.hpp"
```

Comment:
Here and elsewhere: remove the `util/strprintf.hpp` header?

Reply:
No, we need it for the ADL-ized `to_string`.